### PR TITLE
fix(unity-bootstrap-theme): card horizontal fixed height and aspect r…

### PR DESCRIPTION
**Pull request by davidornelas11**
…atio

### Description
Horizontal cards need to have a correct aspect ratio and fixed max-height. Card arrangements also needed to be added as a story to show horizontal cards in the card arrangements

### Links

- [Unity reference site](https://unity.web.asu.edu/@asu/unity-bootstrap-theme/index.html?path=/story/molecules-cards-examples--horizontal)
- [JIRA ticket](https://asudev.jira.com/browse/UDS-1297?atlOrigin=eyJpIjoiZTFjN2EzYjAzODBlNDdmMTg3NGVmOTI0YTU1NjZkMDciLCJwIjoiaiJ9)

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples
- [x] Add/updated READMEs/docs
- [x] No new console errors
- [x] Accessibility checks

### Browsers

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] Edge

### CODE REVIEW

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
@davidornelas11 This is looking good. Like the new templating.  In mobile view with side-by-side horizontal cards, it looks like some work is needed. Could these switch to stacked in mobile?

Screenshot 2023-08-15 at 10 29 00 AM

**Comment by mlsamuelson** [comment]
@davidornelas11 Looks like the mobile stacking happens with how these are implemented in WS2 (see https://dev-webspark-ci.ws.asu.edu/card-arrangement). So, would be nice if that showed in Storybook. Maybe not a requirement if it's a challenge.

**Comment by davidornelas11** [comment]
Not a challenge.; I can add that stacking

**Comment by davidornelas11** [comment]
> @davidornelas11 Looks like the mobile stacking happens with how these are implemented in WS2 (see https://dev-webspark-ci.ws.asu.edu/card-arrangement). So, would be nice if that showed in Storybook. Maybe not a requirement if it's a challenge.

So this most recent change i pushed up involves wrapping the horizontal cards in the card arrangement, as adding more templates using the templates control just adds another component outside the html-root wrapper, so it can not be altered from within the component. If my recent code is too messy, I suggest not having them stack since it is not a card arrangement

File: packages/unity-bootstrap-theme/stories/molecules/cards/cardVariations.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import cardsImage from './cards-image.jpg';
+
+export const horizontalCardsMap = (type = "", args) => {
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cardVariations.js]: change to  ({type = "", numCards=1})

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 2]
```diff
@@ -3,8 +3,20 @@ import { googleAnalytics as initFunc } from '@asu/unity-bootstrap-theme/js/data-
...
+    name: "Columns",
+    options: [1, 2, 3, 4],
+    defaultValue: 1,
+    control: {
+      type: "radio",
+    },
+  }
+};
+
+export default createComponent('Cards', 'Molecules', 'Examples', extraOptions);
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: Remove extraOptions and add it below to the one story that needs it

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 3]
```diff
@@ -56,6 +68,12 @@ export const cardImages = createStory(
     </div>
   </div>
 );
+cardImages.args  = {
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: Remove all instances of this code block
```
.args = {
  template: 1
};
cardTitles.parameters = {
  controls: { exclude: ["Columns"] },
};
```

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 4]
```diff
@@ -977,192 +1037,37 @@ export const cardVariations = createStory(
...
-            </a>
-          </div>
-        </div>
-      </div>
-    </div>
-
+  args => {
+  return (
+    <div>
+    {horizontalCard(null, args)}
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: change to `{horizontalCard({...args, type: null})}`

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 5]
```diff
@@ -977,192 +1037,37 @@ export const cardVariations = createStory(
...
-              data-ga-region="main content"
-              data-ga-section="card default title"
-            >
-              Button link here
-            </a>
-          </div>
-        </div>
-      </div>
-    </div>
+    {horizontalCard("degree", args)}
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: change to `{horizontalCard({...args, type: "degree"})}`

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 6]
```diff
@@ -977,192 +1037,37 @@ export const cardVariations = createStory(
...
-              data-ga-region="main content"
-              data-ga-section="card default title"
-            >
-              Button link here
-            </a>
-          </div>
-        </div>
-      </div>
-    </div>
+    {horizontalCard("event", args)}
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: change to `{horizontalCard({...args, type: "event"})}`

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 7]
```diff
@@ -977,192 +1037,37 @@ export const cardVariations = createStory(
...
-            >
-              Button link here
-            </a>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>,
-  { initFunc }
+    {horizontalCard("story", args)}
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js]: change to `{horizontalCard({...args, type:"story"})}`

File: packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js [hunk 8]
```diff
@@ -977,192 +1037,37 @@ export const cardVariations = createStory(
...
+  );
+  },
+  { initFunc, omitTemplate: false }
 );
+horizontal.args = {
+  template: 1
+};
+horizontal.parameters = {
+  controls: { exclude: ["Template"] },
+}
```
Comment (scott-williams-az) [inline packages/unity-bootstrap-theme/stories/molecules/cards/cards.examples.stories.js:1039]: add the extraOptions here
```
horizontal.argTypes = {
  numCards: {
    name: "Columns",
    options: [1, 2, 3, 4],
    defaultValue: 1,
    control: {
      type: "radio",
    },
  }
}
```

**Review by scott-williams-az — CHANGES_REQUESTED** [review]

**Review by mlsamuelson — APPROVED** [review]

**Comment by mlsamuelson** [comment]
I've approved. Can't be merged until @scott-williams-az also approves since he had requested changes. From what I see, those changes were made.

**Review by scott-williams-az — APPROVED** [review]

**Pull request merged**